tests/conftest.py

```diff
@@ -106,32 +108,32 @@ def respond_invitation(selenium, request_page, url, accept, quota=None, comment=
 
         request_page(selenium, url, by=By.CLASS_NAME, wait_for_element='note_editor')
 
-        container = selenium.find_element_by_class_name('note_editor')
+        container = selenium.find_element(By.CLASS_NAME, 'note_editor')
 
-        buttons = container.find_elements_by_tag_name("button")
+        buttons = container.find_elements(By.TAG_NAME, "button")
         assert len(buttons) == 2
 
         if quota:
             buttons[1].click() ## Decline
             time.sleep(1)
-            reduce_quota_link = selenium.find_element_by_class_name('reduced-load-link')
+            reduce_quota_link = selenium.find_element(By.CLASS_NAME, 'reduced-load-link')
             reduce_quota_link.click()
             time.sleep(1)
-            dropdown = selenium.find_element_by_class_name('dropdown-select__input-container')
+            dropdown = selenium.find_element(By.CLASS_NAME, 'dropdown-select__input-container')
             dropdown.click()
             time.sleep(1)
-            values = selenium.find_elements_by_class_name('dropdown-select__option')
+            values = selenium.find_elements(By.CLASS_NAME, 'dropdown-select__option')
             assert len(values) > 0
             values[0].click()
             time.sleep(1)
-            button = selenium.find_element_by_xpath('//button[text()="Submit"]')
+            button = selenium.find_element(By. XPATH, '//button[text()="Submit"]')
```

Comment:
there's some extra space which looks not right

Reply:
fixed. It is weird the tests are not failing because of this